pkg/virt-operator/install-strategy/create.go

```diff
@@ -85,6 +85,27 @@ func objectMatchesVersion(objectMeta *metav1.ObjectMeta, version string, imageRe
 	return false
 }
 
+func podSpecsMatch(podSpec *corev1.PodSpec, reference *corev1.PodSpec) bool {
+	if !reflect.DeepEqual(podSpec.NodeSelector, reference.NodeSelector) {
+		//debugging cruft
+		log.Log.Infof("node selectors differ")
```

Comment:
just a reminder to remove the debug log lines.

Reply:
ouch lol. glad I left a note. thanks for the reminder